packages/fxa-auth-db-mysql/lib/db/mem.js

```diff
@@ -1086,6 +1086,41 @@ module.exports = function (log, error) {
     }).reverse())
   }
 
+  Memory.prototype.securityEventsByUid = function (uid) {
+    if (! uid) {
+      return P.reject(error.notFound())
+    }
+    const key = uid.toString('hex')
```

Comment:
I would just reassigned uid buffer to uid string here. It makes it a little bit easier to follow the code than using `key`.

```
uid = uid.toString('hex')
```